sklearn/metrics/_pairwise_distances_reduction/meson.build

```diff
@@ -148,6 +148,9 @@ _argkmin_classmode = py.extension_module(
   dependencies: [np_dep],
   override_options: ['cython_language=cpp'],
   cython_args: cython_args,
+  # XXX: for some reason -fno-sized-deallocation is needed otherwise there is
+  # an error with undefined symbol: _ZdlPv at import time in manylinux wheels
+  cpp_args:['-fno-sized-deallocation'],
```

Comment:
Let's open an issue (and add the URL to the comment) to investigate the root cause of this missing symbol in a manylinux env.